app/views/foods/index.html.erb

```diff
@@ -0,0 +1,26 @@
+
+<h1>Foods</h1>
+<h3><%= @count %> RESULTS</h3>
+
+<table id="ingredients">
+  <thead>
+    <tr>
+      <th>Gtin upc code</th>
+      <th>Description</th>
+      <th>Brand</th>
+      <th>Ingredient</th>
+      <th colspan="3"></th>
+    </tr>
+  </thead>
+
+  <tbody>
+    <% @foods.each do |food| %>
+      <tr>
+        <td><%= food[:gtinUpc] %></td>
+        <td><%= food[:lowercaseDescription] %></td>
+        <td><%= food[:brandOwner] %></td>
+        <td><%= food[:ingredients] %></td>
+      </tr>
+    <% end %>
+  </tbody>
+</table>
```

Comment:
I like this setup, but I don't want you to spend extra time on a timed assessment formatting the view unless the spec asks for that.